File: model/src/main/java/net/artux/pda/model/quest/story/StoryStateModel.java
package net.artux.pda.model.quest.story;

import java.io.Serializable;

import lombok.Data;

@Data
public class StoryStateModel implements Serializable {

    private boolean current;
    private boolean over;

    private int storyId;
    private int chapterId;
    private int stageId;

    @Override
    public String toString() {
        return "StoryStateModel{" +
                "current=" + current +
                ", over=" + over +
                ", storyId=" + storyId +
                ", chapterId=" + chapterId +
                ", stageId=" + stageId +
                '}';
    }
}

File: model/src/main/java/net/artux/pda/model/quest/story/StoryStateModel.kt
package net.artux.pda.model.quest.story

import java.io.Serializable

class StoryStateModel : Serializable {
    var current = false
    var over = false
    var storyId = 0
    var chapterId = 0
    var stageId = 0

    override fun toString(): String {
        return "StoryStateModel{" +
                "current=" + current +
                ", over=" + over +
                ", storyId=" + storyId +
                ", chapterId=" + chapterId +
                ", stageId=" + stageId +
                '}'
    }
}